### PR TITLE
Add admin interface for CustomFinders

### LINKED ISSUE #274: Update links to external polling station finders (inc. removing link to LondonElects)
The London Elects polling station finder is now offline, so we should revert to providing London residents with the contact details of their local council.

### PR DESCRIPTION
**Pull request by symroe**
Fixes #274

### CODE REVIEW

File: polling_stations/apps/pollingstations/models.py [hunk 1]
```diff
@@ -144,42 +144,46 @@ def get_custom_finder(self, gss_codes, postcode):
...
+    record.save()
+    """
+    area_code = models.CharField(max_length=9, primary_key=True,
+        help_text="The GSS code for this area")
+    base_url = models.CharField(blank=True, max_length=255,
+        help_text="The landing page for the polling station finder")
+    can_pass_postcode = models.BooleanField(default=False,
+        help_text="Does the URL have '?postcode=' in it?")
+    message = models.TextField(blank=True,
+        default="This council has it's own polling station finder:")
```
Comment (andylolz) [inline polling_stations/apps/pollingstations/models.py]: superfluous apostrophe here

File: polling_stations/apps/pollingstations/models.py [hunk 2]
```diff
@@ -144,42 +144,46 @@ def get_custom_finder(self, gss_codes, postcode):
...
+    and/or a message that we might want to show.
+
+
+    Example content:
+
+    record = CustomFinder(
+        area_code='E07000082'
+        base_url='https://stroud.maps.arcgis.com/apps/webappviewer/index.html?id=ea6bf4b3655542c1a05c8d7e87d32bb1'
+        can_pass_postcode=False
+        message="Stroud District Council has its' own polling station finder:"
```
Comment (andylolz) [inline polling_stations/apps/pollingstations/models.py]: superfluous apostrophe here

File: polling_stations/apps/pollingstations/models.py [hunk 3]
```diff
@@ -144,42 +144,46 @@ def get_custom_finder(self, gss_codes, postcode):
...
+        can_pass_postcode=False
+        message='<h2>We're working on it!</h2>Ceredigion Council have provided polling station data. It will be available soon.'
+    )
+    record.save()
+
+    record = CustomFinder(
+        area_code='N07000001'
+        base_url='http://www.eoni.org.uk/Offices/Postcode-Search-Results?postcode='
+        can_pass_postcode=True
+        message='The Electoral Office of Northern Ireland has its' own polling station finder:'
```
Comment (andylolz) [inline polling_stations/apps/pollingstations/models.py]: superfluous apostrophe here

**Comment by andylolz** [comment]
Typos flagged up. But sooner this can be merged, the better! Getting #274 fixed would be awesome.

**Comment by symroe** [comment]
Thanks Andy, typos fixed, merged

**Pull request merged**